src/utils/coordinates/coordinateExtractors.js

```diff
@@ -3,12 +3,24 @@ import proj4 from 'proj4'
 import { LV95, WGS84 } from '@/utils/coordinates/coordinateSystems'
 import { reprojectUnknownSrsCoordsToWGS84 } from '@/utils/coordinates/coordinateUtils'
 import { toPoint as mgrsToWGS84 } from '@/utils/militaryGridProjection'
+import { isNumber } from '@/utils/numberUtils.js'
 
 // 47.5 7.5
 const REGEX_WEB_MERCATOR = /^\s*([\d]{1,3}[.\d]+)\s*[ ,/]+\s*([\d]{1,3}[.\d]+)\s*$/i
+// 47.5N 7.5E
+const REGEX_WEB_MERCATOR_WITH_CARDINALS =
+    /^\s*([\d]{1,3}[.\d]+)\s*([NSEW]?)\s*[ ,/]+\s*([\d]{1,3}[.\d]+)\s*([NSEW]?)\s*$/i
```

Comment:
Should be 
```
/^\s*(\d{1,3}(\.\d+)?)\s*([NSEW]?)\s*[,/]?\s*(\d{1,3}(\.\d+)?))\s*([NSEW]?)\s*$/i
```
But are you sure that web mercator cardinal are not more than 999 ? Because your regex specify 3 digit, so the regex above would now match 999.99999999999 N but not 1000 N